index.html

```diff
@@ -140,16 +155,17 @@ <h3 class="project-title"> Multi-Post Stories <br> Gain+Glory</h3>
             </div>
         </section>
         <section class="heading-text about-me">
-              <h2 class="primary color1">About me</h2>  
-              <p class="supporting-text"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
-                don’t hesitate to contact me.
-              </p>
-              <div class="btn-container">
-                <button class="resume-btn">Get my resume</button>
-              </div>
+            <div class="about-me-container">
+                <h2 class="primary color1">About me</h2>  
+                <p class="supporting-text"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don’t hesitate to contact me.</p>
+                <div class="btn-container">
+                    <button class="resume-btn">Get my resume</button>
+                </div>
+            </div>  
```

Comment:
- [ ] kindly consider making sure that the two background images in this section are all visible on this section to make your design meet the design requirements check the images below to see the difference 
- your design 👇 
<img width="1407" alt="Screen Shot 2023-06-17 at 23 10 08" src="https://github.com/BHObserver/portfolio/assets/60146030/860b315d-8ae6-4868-9fa0-ad1451bf120d">

- Figma design 👇 
<img width="788" alt="Screen Shot 2023-06-17 at 23 11 59" src="https://github.com/BHObserver/portfolio/assets/60146030/78a989ce-7361-4e40-b425-ee1b6fd6f49b">

Reply:
I am sorry, could you tell me in which width size is it not working?